src/runboat/k8s.py

```diff
@@ -153,6 +152,7 @@ class DeploymentVars(BaseModel):
     build_env: dict[str, str]
     build_secret_env: dict[str, str]
     build_template_vars: dict[str, str]
+    kubefiles_path: Path
```

Comment:
DeploymentVars is the context of the jinja template rendering. So this `kubefiles_path` does not really belong here.
Can you try passing as a new argument it to `deploy()` and `_render_kubefiles()` and see how it fits?

Reply:
Tried in last commit, makes more sense indeed